autoscale/src/main/java/com/sequenceiq/periscope/domain/LoadAlertConfiguration.java

```diff
@@ -35,8 +45,22 @@ public Integer getCoolDownMinutes() {
     }
 
     @JsonIgnore
-    public Long getCoolDownMillis() {
-        return TimeUnit.MILLISECONDS.convert(coolDownMinutes, TimeUnit.MINUTES);
+    public Long getPollingCoolDownMillis() {
+        Integer pollingCoolDown = coolDownMinutes != null ? coolDownMinutes :
+                Math.min(scaleUpCoolDownMinutes, scaleDownCoolDownMinutes);
+        return TimeUnit.MILLISECONDS.convert(pollingCoolDown, TimeUnit.MINUTES);
+    }
+
+    @JsonIgnore
+    public Long getScaleUpCoolDownMillis() {
+        Integer scaleUpCoolDown = coolDownMinutes != null ? coolDownMinutes : scaleUpCoolDownMinutes;
```

Comment:
I think this should be the opposite. If scaleUpCoolDownMinutes is set - use it, otherwise fallback to the default.
(Similarly for the scale-down)

i.e. A user can set either the scale-up or the scale-down config to override the default value for that specific scenario.

Would apply to getPollingCooldownMillis as well.

Reply:
ok